nuxt.config.js

```diff
@@ -12,15 +12,30 @@ export default {
                     authorization: `Token ${process.env.AUTH_TOKEN}`,
                 },
             }
-            const talks = await axios.get(
-                `${DEFAULT_BASE_URL}/api/events/speeches/?event_types=talk,sponsored`,
-                config,
-            )
-            const tutorials = await axios.get(
-                `${DEFAULT_BASE_URL}/api/events/speeches/?event_types=tutorial`,
-                config,
-            )
+            const talks = await axios
+                .get(
+                    `${DEFAULT_BASE_URL}/api/events/speeches/?event_types=talk,sponsored`,
+                    config,
+                )
+                .catch((error) => {
+                    if (error.response.status === 404) {
+                        return { data: [] }
+                    }
+                })
+            const tutorials = await axios
+                .get(
+                    `${DEFAULT_BASE_URL}/api/events/speeches/?event_types=tutorial`,
+                    config,
+                )
+                .catch((error) => {
+                    if (error.response.status === 404) {
+                        return { data: [] }
+                    }
+                })
             const getAllDetailTalks = async () => {
+                if (!talks.data) {
+                    return []
+                }
```

Comment:
if we can guarantee that talks.data always return array (includes empty array),
could we remove this condition?

Reply:
yes, currently is not quite stable so that the response would get 500. Maybe I could add #FIXME here so that we could remove it in the future.